src/js/components/Anchor/StyledAnchor.js

```diff
@@ -79,6 +79,7 @@ const StyledAnchor = styled.a.withConfig({
   ${(props) => props.disabled && disabledStyle}
   ${(props) => props.focus && focusStyle()}
   ${(props) => props.theme.anchor.extend}
+  display: inline-flex;
```

Comment:
@umeshiscreative This looks great. To follow with our conventions, can you place this `display: inline-flex;` just below the box-sizing: border-box on line 35? Thank you.